src/utils/tools.ts

```diff
@@ -2,17 +2,17 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as util from 'util';
 import { createElement, fromAskScriptAst } from '../askjsx';
-import { parseToAst } from '../askscript';
+import { parseToAst, AskScriptCode } from '../askscript';
 import { Options, resources, runUntyped } from '../askvm';
 
 const myLogger = util.debuglog('');
 
 export async function e2e(
-  script: string,
+  askScriptCode: AskScriptCode,
```

Comment:
this is the same what @mhagmajer said "out of the scope of this PR"

What the code should does in this PR?

Reply:
@km4 @mhagmajer , this is a direct result of mhagmajer's request to change string to AskScriptCode [in this comment](https://github.com/xFAANG/askql/pull/114#discussion_r439779775).